doc/api/module.md

```diff
@@ -1587,6 +1587,43 @@ import { findSourceMap, SourceMap } from 'node:module';
 const { findSourceMap, SourceMap } = require('node:module');
 ```
 
+### `module.setSourceMapsSupport(enabled[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `enabled` {boolean} Enable the source map support.
+* `options` {Object} Optional
+  * `nodeModules` {boolean} If enabling the support for files in `node_modules`.
+  * `generatedCode` {boolean} If enabling the support for generated code from `eval` or `new Function`.
```

Comment:
It's missing doc for default values I think

Reply:
Thanks, updated!